spec/system/candidate_interface/candidate_withdraws_spec.rb

```diff
@@ -56,4 +60,16 @@ def then_my_application_should_be_withdrawn
   def and_a_slack_notification_is_sent
     expect_slack_message_with_text 'has withdrawn their application'
   end
+
+  def when_i_try_to_visit_the_withdraw_page
+    visit candidate_interface_withdraw_path(id: @application_choice.id)
+  end
+
+  def then_i_can_not_see_the_withdaw_page
+    expect(page).not_to have_content('Confirm you would like to withdraw your application to study')
```

Comment:
Test for 404 instead of absence of a text (the text might change).

Reply:
The acceptance criteria suggest the following:  `They should be redirected to the application dashboard when a candidate attempts to access these pages.`
I changed the assertion to explicitly test for `application dashboard`. I'm also happy to change the behaviour and redirect to 404 if you think that's better!

Reply:
Yes 404 seems better!